extensions/components/com_redcore/admin/views/webservice/tmpl/default.php

```diff
@@ -14,150 +14,12 @@
 JHtml::_('behavior.keepalive');
 JHtml::_('rbootstrap.tooltip');
 JHtml::_('rjquery.chosen', 'select');
+JHtml::script(JUri::root() . '/media/redcore/js/redcore.js');
```

Comment:
btw I dont see you including that file in this PR... from your command prompt please go to: "cd build" and then run "gulp"